neuhatch/utils.py

```diff
@@ -3,8 +3,9 @@
 from flask.ext.login import logout_user, logout_user, login_required, current_user
 from neuhatch import config, app, db, login_manager
 
-def get_base_auth():
-    return tweepy.OAuthHandler(config.consumer_key, config.consumer_secret)
+def get_base_auth(callback_url = None):
+    return tweepy.OAuthHandler(
+        config.consumer_key, config.consumer_secret, callback_url)
```

Comment:
Fixed this in 2e4aa9654b5b61ca7d9406338849e8b08b15b249 to set the right argument to the OAuthHandler constructor. Works if callback is None (or not set).